### PR TITLE
Add SSO section for M365 Apps messaging extensions tutorial

### PR DESCRIPTION
**Pull request by erikadoyle**
Added guidance for enabling messaging extensions SSO now that it's supported in Outlook.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4dfe622f858696b6b80ab9ccc3d40bac1eba2374/msteams-platform/m365-apps/extend-m365-teams-message-extension.md) | 90 | 37 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b8f1845f-eec6-4e7b-8d76-96d04df92bda) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/extend-m365-teams-message-extension.md [hunk 1]
```diff
@@ -86,6 +86,28 @@ For users to interact with your messaging extension from Outlook, you'll need to
...
 
+## Update Azure AD app registration for SSO
+
+> [!NOTE]
+> You can skip this step if you are using the [Teams messaging extension search sample](https://github.com/microsoft/BotBuilder-Samples/tree/main/samples/javascript_nodejs/50.teams-messaging-extensions-search), as the scenario doesn't involve Azure Active Directory (AAD) Single Sign-On authentication.
+
+Azure Active Directory Single-sign on (SSO) for messaging extensions works the same way in Outlook [as it does in Teams](/microsoftteams/platform/bots/how-to/authentication/auth-aad-sso-bots), however you will need to add several client application identifiers to the Azure AD app registration of your bot in your tenant's *App registrations* portal.
+
+1. Sign in to [Azure portal](https://portal.azure.com) with your sandbox tenant account.
+1. Open the **App registrations** blade.
```
Comment (v-abirade) [inline msteams-platform/m365-apps/extend-m365-teams-message-extension.md]: Open **App registrations**.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fb886b93f913450c3d3448af88387804ae8827f7/msteams-platform/m365-apps/extend-m365-teams-message-extension.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8c0d79f0-e6ae-4698-b7b7-40ae7f4a7358) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approved

**Pull request merged**